### ISSUE TITLE
[Model Catalog Explorer] Explanation of scientific variables

### ISSUE THREAD

**Issue by dgarijo** [mintproject/mint-ui-lit#84]
Right now we only provide a link to the external variable in SVO. However we do have some metadata explaining what the variable is composed of. We should show it in the GUI. It may require a query per variable though.

**Comment by hvarg** [comment]
We have a way to do this now?

**Comment by dgarijo** [comment]
yes. The description of the each variable has been updated as an html snippet. It is returned in the APi
